public/data/Ratheshan03.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Ratheshan Sathiyamoorthy",
+  "bio": "BSc (Hons) Computer Science Undergraduate | Full Stack Software developer | Open Source Enthusiast",
+  "avatar": "https://github.com/Ratheshan03.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Ratheshan03",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ratheshan-sathiyamoorthy-3aa2891b9/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Ratheshan_03",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/ratheshan_03/",
+      "icon": "instagram"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "June 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
```

Comment:
Here you can provide more description about what you did when you started freelancing. 🙂
Note that it's not obligatory and it can be kept as-is. Providing more information is better, IMO.

Reply:
@Panquesito7 Sorry didn't saw that attribute! I have added now!
Thankyou.